public/desktop.ini

```diff
@@ -0,0 +1,2 @@
+[LocalizedFileNames]
```

Comment:
is this file autogenerated by windows maybe? we don't actually need it in the repo

Reply:
Yeah think so, I'll remove it real quick